test/algorithms/Jamfile

```diff
@@ -25,8 +25,6 @@ test-suite boost-geometry-algorithms
     [ run comparable_distance.cpp      : : : : algorithms_comparable_distance ]
     [ run convert.cpp                  : : : : algorithms_convert ]
     [ run convert_multi.cpp            : : : : algorithms_convert_multi ]
-    [ run convex_hull.cpp              : : : : algorithms_convex_hull ]
-    [ run convex_hull_multi.cpp        : : : : algorithms_convex_hull_multi ]
     [ run correct.cpp                  : : : : algorithms_correct ]
```

Comment:
So it's gone? Will it be fixed later?

Reply:
No, they are moved in a `convex_hull` directory, see https://github.com/boostorg/geometry/pull/756/commits/eae06f70dfb2f499810564a636039ec6474c10ac

Reply:
> 
> 
> No, they are moved in a `convex_hull` directory, see [eae06f7](https://github.com/boostorg/geometry/commit/eae06f70dfb2f499810564a636039ec6474c10ac)

I remember now, the big PR.